pkg/archive/archive_test.go

```diff
@@ -305,7 +305,9 @@ func TestUntarPathWithInvalidSrc(t *testing.T) {
 }
 
 func TestUntarPath(t *testing.T) {
-	skip.If(t, os.Getuid() != 0, "skipping test that requires root")
+	if runtime.GOOS != "windows" {
+		skip.If(t, os.Getuid() != 0, "skipping test that requires root")
```

Comment:
Maybe you can use something like

```go
skip.If(t, runtime.GOOS != "windows" && os.Getuid() != 0, "requires root")
```

The whole idea of using `skip.If()` is to have a one-liner; if we don't want that, it would be better to write it without using `skip.If()`:
```go
if runtime.GOOS != "windows" && os.Getuid() != 0 {
	t.Skip("requires root")
}
```

Choose either way :)

Reply:
same ^^^ for other similar places

Reply:
Ah, yes, that's clearer. Will update.